src/orchestrator/abstracts/ModuleManagerBase_v1.sol

```diff
@@ -1,32 +1,33 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity 0.8.23;
 
+// Interfaces
+import {IModuleManagerBase_v1} from
+    "src/orchestrator/interfaces/IModuleManagerBase_v1.sol";
+
 //External Dependencies
 import {ERC2771Context} from "@oz/metatx/ERC2771Context.sol";
 import {Initializable} from "@oz-up/proxy/utils/Initializable.sol";
 import {ERC165} from "@oz/utils/introspection/ERC165.sol";
 
-// Interfaces
-import {IModuleManager} from "src/orchestrator/base/IModuleManager.sol";
-
 /**
- * @title Module Manager
+ * @title   ModuleManagerBase_v1: Module Manager Base v1 for Inverter Network
  *
- * @dev A contract to manage modules that can execute transactions via this
- *      contract and manage own role-based access control mechanisms.
+ * @dev     A contract to manage modules that can execute transactions via this
```

Comment:
A contract to manage Inverter Network modules.
It allows for adding and removing modules in a local registry for reference.
Additional functionality includes the execution of calls from this contract.